src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,117 @@
+package com.nutomic.syncthingandroid.syncthing;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v4.app.NotificationCompat;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.MainActivity;
+
+/**
+ * Broadcast-receiver to control and configure SyncThing remotely
+ *
+ * Created by sqrt-1764 on 25.03.16.
+ */
+public class AppConfigReceiver extends BroadcastReceiver {
+    private static final int ID_NOTIFICATION_BACKGROUND_ACTIVE = 2;
+
+    public static final String ACTION_START = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_START";
+    public static final String ACTION_STOP  = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_STOP";
+    public static final String ACTION_STOP2 = "com.nutomic.syncthingandroid.action.ACTION_SYNCTHING_STOP2";
```

Comment:
That's a good point, especially seeing as this is a public interface (the other intents/actions aren't). But both "action" and syncthing appear twice in the string. I would just go with `com.nutomic.syncthingandroid.action.START` then.